pkg/awsManager/s3Manager.go

```diff
@@ -43,17 +43,12 @@ func DeleteS3Buckets(client clientpkg.Client, s3BucketsToBeDeleted []*string, lo
 		// Deleting the bucket
 		_, err := client.DeleteBucket(&s3.DeleteBucketInput{Bucket: bucket})
 		if err != nil {
-			if err, ok := err.(awserr.Error); ok {
-				switch err.Code() {
-				default:
-					logger.Error(err, "could not delete bucket", bucket)
-					s3BucketsNotDeleted = append(s3BucketsNotDeleted, bucket)
-				}
-			} else {
-				logger.Error(err, "could not delete bucket", *bucket)
-				s3BucketsNotDeleted = append(s3BucketsNotDeleted, bucket)
-			}
```

Comment:
I wasn't aware it made the logs any cleaner.  I removed this just because it looked like duplicated code with no purpose.  But I guess if we do cast it we'd also get the Code.  🤔